src/gui/hover.c

```diff
@@ -5,7 +5,7 @@
  *
  * Displays mouse-over (hover) texts.
  *
- * Add "log_char(cn,LOG_SYSTEM,0,"�c5.");" to the very end of int look_item() in tool.c!
+ * Add "log_char(cn,LOG_SYSTEM,0,"�c5.");" to the very end of int look_item() in tool.c!

```

Comment:
Out of curiousity, what symbol is this? Won't show in github. Same with the other lines where the only change is this undisplayable character. Is it that degree symbol that's all over the codebase? If so, that's fine - in my branch (and I think even in one of your commits) these got fixed to the literal byte code. Just wanted to double check since I can't tell in this review.

If it's literally identical, then theese lines should probably be taken out of the commit, as they are spurious updates (maybe even automatic by your editor)